algorithms/facenet/vectorize.py

```diff
@@ -0,0 +1,47 @@
+import json
+import os
+from typing import List
+
+import numpy as np
+import tensorflow as tf
+from facenet_sandberg import face
+
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+tf.logging.set_verbosity(tf.logging.ERROR)
+
+FaceVector = List[float]
+Image = np.array
+
+
+def get_face_vectors(img_path: str, prealigned: bool) -> List[List[float]]:
```

Comment:
Nit: the return type can now be `List[FaceVector]`

Reply:
fixed!